src/gui/include/gui/gui.h

```diff
@@ -333,7 +333,7 @@ class Descriptor
     ActionCallback callback;
   };
   using Actions = std::vector<Action>;
-  static constexpr std::string_view deselect_action_ = "deselect";
+  static constexpr std::string_view kDeselectAction = "deselect";
```

Comment:
are we no longer doing trailing `_` for class members?

Reply:
https://google.github.io/styleguide/cppguide.html#Variable_Names

```
Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore. The exception to this is static constant class members, which should follow the rules for [naming constants](https://google.github.io/styleguide/cppguide.html#Constant_Names).
```

This is the exception case.

Reply:
I got started on this by finding a couple of places where the trailing _ was missing (not an exception, just an error).